cypress/integration/userCanLogout.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('User Log-Out', () => {
+   beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json"
+    })
+  });
```

Comment:
General question, do  we need a PR with a request spec that verifies that logout has occurred?

Reply:
You can make one here to make sure the browser doesn't have anything in localStorage after the page is reloaded